script.js

```diff
@@ -1,4 +1,112 @@
+var die1;
+var die2;
+var player1Score = [];
+var player1GlobalScore = 0;
+var player2Score = [];
+var player2GlobalScore = 0;
+var player = 1;
+var mode = "dice roll";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var outputMessage;
+  while (player == 1) {
+    if (mode == "dice roll") {
+      var diceResult = roll2Dice();
+      mode = "choose dice order";
+      outputMessage = `Welcome Player 1. <br>
```

Comment:
I see that you have several of these outputMessages. One thing you could thing is to create a helper function that generates a custom message for you depending on the game mode and player. e.g. generateMessage(gameMode, player)